lib/tasks/deploy.rake

```diff
@@ -134,12 +134,19 @@ namespace :deploy do
   end
 
   task :rolling_deploy do
```

Comment:
Would be good to validate that the contents of `rolling_deploy_on_failure` are one of the expected options.